src/app/products/products.module.ts

```diff
@@ -74,6 +74,7 @@ import { EditChargeComponent } from './charges/edit-charge/edit-charge.component
 import { MatFormFieldModule, MatSelectModule, MatOptionModule } from '@angular/material';
 import { FormsModule, ReactiveFormsModule } from '@angular/forms';
 
+import { DividendsComponent } from './share-products/dividends-share-product/dividends.components';
```

Comment:
Rename this component as `ShareProductsDividendsComponent`, import it where other ShareProduct imports are.